test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/DeferWithLabelTestSnapshot.kt

```diff
@@ -0,0 +1,102 @@
+// @formatter:off
+package graphql.nadel.tests.next.fixtures.defer
+
+import graphql.nadel.tests.next.ExpectedNadelResponse
+import graphql.nadel.tests.next.ExpectedServiceCall
+import graphql.nadel.tests.next.TestSnapshot
+import graphql.nadel.tests.next.listOfJsonStrings
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.listOf
+
+/**
+ * This class is generated. Do NOT modify.
+ *
+ * Refer to [graphql.nadel.tests.next.UpdateTestSnapshots
+ */
+@Suppress("unused")
+public class DeferWithLabelTestSnapshot : TestSnapshot() {
+    override val calls: List<ExpectedServiceCall> = listOf(
+            ExpectedServiceCall(
+                service = "defer",
+                query = """
+                | {
+                |   defer {
+                |     hello
+                |     ... @defer(label: "slow-defer") {
+                |       slow
+                |     }
+                |   }
+                | }
+                """.trimMargin(),
+                variables = "{}",
+                response = """
+                | {
+                |   "defer": {
+                |     "hello": "helloString"
```

Comment:
The initial response should contain the `hasNext` field with value `true`. See example here:
https://github.com/graphql/graphql-over-http/blob/main/rfcs/IncrementalDelivery.md#content-type-multipartmixed

Reply:
Hmm that's probably my fault. This is odd, it doesn't even contain the `data` field.